velox/dwio/common/tests/utils/FilterGenerator.cpp

```diff
@@ -147,17 +150,34 @@ std::unique_ptr<Filter> ColumnStats<float>::makeRangeFilter(
 template <>
 std::unique_ptr<Filter> ColumnStats<double>::makeRangeFilter(
     float startPct,
-    float selectPct) {
+    float selectPct,
+    FilterSpec filterSpec) {
   if (values_.empty()) {
-    return std::make_unique<velox::common::IsNull>();
+    if (filterSpec.allowNulls_) {
+      return std::make_unique<velox::common::IsNull>();
+    } else {
```

Comment:
Does it matter to return different filters according to allowNulls_ when the values are empty? It looks unnecessary for me.

Reply:
Well. It's `IsNull` filter now, which returns true on nulls. But it's true that I can merge them